assets/dev/scss/frontend/conditionals/e-swiper.scss

```diff
@@ -84,7 +83,7 @@
 		}
 
 		// `:where()` is used to decrease the specificity of the selectors.
-		:where( .swiper-container-horizontal ) {
+		:where( .swiper-horizontal ) {
```

Comment:
@rami-elementor Have you tested this aspect of the code? Was there a bug in the previous version? Should it always have been `.swiper-container-horizontal, .swiper-horizontal`.

I believe we should test this manually.

Would you like me to test this?

Reply:
Yes please.

Reply:
Your code seems to be correct.